modules/data-streams/src/javaRestTest/java/org/elasticsearch/datastreams/logsdb/qa/StandardVersusLogsIndexModeRandomDataChallengeRestIT.java

```diff
@@ -36,12 +38,14 @@
  */
 public class StandardVersusLogsIndexModeRandomDataChallengeRestIT extends StandardVersusLogsIndexModeChallengeRestIT {
     private final ObjectMapper.Subobjects subobjects;
+    private final boolean storeArraySource;
 
     private final DataGenerator dataGenerator;
 
     public StandardVersusLogsIndexModeRandomDataChallengeRestIT() {
         super();
         this.subobjects = randomFrom(ObjectMapper.Subobjects.values());
+        this.storeArraySource = usually();
```

Comment:
I would just do `randomBoolean` here. In my experience `usually` is very frequent.